drivers/src/sha384.rs

```diff
@@ -57,6 +86,63 @@ impl Sha384 {
         Ok(op)
     }
 
+    /// Update the digest with data
+    ///
+    /// # Arguments
+    ///
+    /// * `data` - Data to used to update the digest
+    pub fn update_ctx(&mut self, ctx: &mut Sha384Ctx, data: &[u8]) -> CaliptraResult<()> {
```

Comment:
These implementation are mostly copied from the Op-API. If we want I can update that API to use these. I just didn't want to mess too much with driver functions that ROM is using.

Reply:
For code-size in binaries that use both APIs, it makes sense to combine. But like you say there may be some pushback there....